src/Files.App/Helpers/DynamicDialogFactory.cs

```diff
@@ -224,5 +224,70 @@ public static DynamicDialog GetFor_CredentialEntryDialog(string path)
 
 			return dialog;
 		}
+
+		public static DynamicDialog GetFor_GitCheckoutConflicts()
+		{
+			DynamicDialog dialog = null!;
+
+			var bringChangesRadio = new RadioButton()
+			{
+				Content = "BringChanges".GetLocalizedResource(),
+				IsChecked = true,
+			};
+			var stashChangesRadio = new RadioButton()
+			{
+				Content = "StashChanges".GetLocalizedResource(),
+			};
+			var discardChangesRadio = new RadioButton()
+			{
+				Content = "DiscardChanges".GetLocalizedResource(),
+			};
+
+			bringChangesRadio.Checked += (button, args) =>
+			{
+				dialog.ViewModel.AdditionalData = GitCheckoutOptions.BringChanges;
+			};
+			stashChangesRadio.Checked += (button, args) =>
+			{
+				dialog.ViewModel.AdditionalData = GitCheckoutOptions.StashChanges;
+			};
+			discardChangesRadio.Checked += (button, args) =>
+			{
+				dialog.ViewModel.AdditionalData = GitCheckoutOptions.DiscardChanges;
+			};
```

Comment:
Can you try a ListView instead?

Reply:
That would be a better alternative

Reply:
I updated the picture in the description.
Anyway, I'm wondering if we have to pre-select the first option...